### PR TITLE
Our projects component (fixes #12)

### LINKED ISSUE #12: Our projects component

### PR DESCRIPTION
**Pull request by OtisRed**
In this PR I have pulled only rendering the list of projects and hover effects. There is some more work needed to make the items clickable buttons that lead to modal content with the description of each project but I will do it in a separate task.

To test this PR you need to also pull #101 with updated Veutify - otherwise there is a leftover bug that messes v-hover effects.

### CODE REVIEW

File: frontend/src/components/OurProjects.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    align-items: center;
+    bottom: 0;
+    justify-content: center;
+    opacity: .9;
+    position: absolute;
+    width: 100%;
+    background: #0CAEE7;
+    color: white;
+  }
+</style>
```
Comment (Bosmanfrx) [inline frontend/src/components/OurProjects.vue]: newline

File: frontend/src/components/OurProjects/OurProjects.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                  <v-img :src="hoveredImg" max-width="50%"></v-img>
+                </div>
+              </v-expand-transition>
+            </v-card>
+          </v-hover>
+        </v-flex>
+      </v-layout>
+    </v-container>
+  </v-item-group>
+</div>
```
Comment (Bosmanfrx) [inline frontend/src/components/OurProjects/OurProjects.html]: ditto

File: frontend/src/components/OurProjects.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-item-group>
+    <v-container grid-list>
+      <v-layout row wrap>
+        <v-flex id="items" v-for="project in projects" xs12 md6 lg4 xl3>
+          <v-hover>
+            <v-card slot-scope="{ hover }">
+              <v-img :src="project.image" aspect-ratio="1.9"></v-img>
+              <v-expand-transition>
+                <div v-if="hover" id="card--reveal" style="height: 100%">
```
Comment (Loczi94) [inline frontend/src/components/OurProjects.vue]: Styles shouldn't be defined inline.

File: frontend/src/components/OurProjects.vue [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                  <v-img :src="hoveredImg" max-width="50%"></v-img>
```
Comment (Loczi94) [inline frontend/src/components/OurProjects.vue]: Styles shouldn't be defined inline.

**Pull request merged**